src/azure-cli-core/azure/cli/core/extension/tests/latest/test_extension_commands.py

```diff
@@ -397,6 +397,69 @@ def test_list_available_extensions_incompatible_cli_version(self):
             self.assertIsInstance(res, list)
             self.assertEqual(len(res), 0)
 
+    def test_add_extension_install_setup_extras(self):
+        """
+        Tests extension addition while specifying --install-setup-extras parameter.
```

Comment:
`--install-setup-extras` is not defined.

Reply:
fixed